src/Compilers/CSharp/Portable/Symbols/SignatureOnlyParameterSymbol.cs

```diff
@@ -87,5 +83,31 @@ public override bool IsImplicitlyDeclared
         internal override ModuleSymbol ContainingModule { get { throw ExceptionUtilities.Unreachable; } }
 
         #endregion Not used by MethodSignatureComparer
+
+        public override bool Equals(object obj)
```

Comment:
It isn't clear that this is symmetric. How do you know some other non-`SignatureOnlyParameterSymbol` won't say it is equal to a `SignatureOnlyParameterSymbol`?